apps/chat/src/components/Common/Modal.tsx

```diff
@@ -21,28 +21,36 @@ import {
 
 import classNames from 'classnames';
 
+import { ModalState } from '@/src/types/modal';
+
+import { Spinner } from './Spinner';
+import Tooltip from './Tooltip';
+
 export interface Props extends FormHTMLAttributes<HTMLFormElement> {
-  portalId: string;
-  isOpen: boolean;
-  onClose: () => void;
   children: ReactNode | ReactNode[];
+  portalId: string;
+  state?: ModalState;
```

Comment:
`isOpen: boolean;` was better

Reply:
Not sure, because we need to support loading state and having 2 same purpose props not needed as for me